index.js

```diff
@@ -1,17 +1,37 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    (this.items = []), (this.length = this.items.length);
```

Comment:
I have literally, never seen this before. But if it works, thank you for showing me something new.
🤷‍♂️

Reply:
That was auto-formatted by Pritter. It passed the tests and I thought to leave it there :)